swarm/src/lib.rs

```diff
@@ -1018,88 +1018,92 @@ where
         // across a `Deref`.
         let this = &mut *self;
 
+        // This loop polls the components below in a prioritized order.
+        //
+        // 1. [`NetworkBehaviour`]
```

Comment:
Nit: Unless you are in rustdoc land (`///`), I don't think this reference will do anything.

Reply:
No feeling strongly at all, just figured it might have been a mistake :)